finders/metadata/employment-appeal-tribunal-decisions.json

```diff
@@ -0,0 +1,284 @@
+    },
+    {
+      "key": "right-to-be-accompanied",
+      "radio_button_name": "Right To Be Accompanied",
+      "topic_name": "right to be accompanied",
+      "prechecked": false
+    },
+    {
+      "key": "rights-on-insolvency",
+      "radio_button_name": "Rights on Insolvency",
+      "topic_name": "rights on insolvency",
+      "prechecked": false
+    },
+    {
+      "key": "sex-discrimination",
+      "radio_button_name": "Sex Discrimination",
+      "topic_name": "sex discrimination",
+      "prechecked": false
+    },
+    {
+      "key": "sexual-orientation-discrimination-transexualism",
+      "radio_button_name": "Sexual Orientation Discrimination/Transexualism",
+      "topic_name": "sexual orientation discrimination transexualism",
+      "prechecked": false
+    },
+    {
+      "key": "statutory-discipline-and-grievance-procedures",
+      "radio_button_name": "Statutory Discipline and Grievance Procedures",
+      "topic_name": "statutory discipline and grievance procedures",
+      "prechecked": false
+    },
+    {
+      "key": "time-limits-obsolete-topic",
+      "radio_button_name": "Time Limits (obsolete topic)",
+      "topic_name": "time limits obsolete topic",
+      "prechecked": false
+    },
+    {
+      "key": "time-off",
+      "radio_button_name": "Time Off",
+      "topic_name": "time off",
+      "prechecked": false
+    },
+    {
+      "key": "trade-union-membership",
+      "radio_button_name": "Trade Union Membership",
+      "topic_name": "trade union membership",
+      "prechecked": false
+    },
+    {
+      "key": "trade-union-rights",
+      "radio_button_name": "Trade Union Rights",
+      "topic_name": "trade union rights",
+      "prechecked": false
+    },
+    {
+      "key": "transfer-of-undertakings",
+      "radio_button_name": "Transfer of Undertakings",
+      "topic_name": "transfer of undertakings",
+      "prechecked": false
+    },
+    {
+      "key": "unfair-dismissal",
+      "radio_button_name": "Unfair Dismissal",
+      "topic_name": "unfair dismissal",
+      "prechecked": false
+    },
+    {
+      "key": "unlawful-deduction-from-wages",
+      "radio_button_name": "Unlawful Deduction from Wages",
+      "topic_name": "unlawful deduction from wages",
+      "prechecked": false
+    },
+    {
+      "key": "victimisation-discrimination",
+      "radio_button_name": "Victimisation Discrimination",
+      "topic_name": "victimisation discrimination",
+      "prechecked": false
+    },
+    {
+      "key": "working-time-regulations",
+      "radio_button_name": "Working Time Regulations",
+      "topic_name": "working time regulations",
+      "prechecked": false
+    }
+  ],
+  "summary": "<p>Find decisions on appeals against employment tribunals heard by the Employment Appeal Tribunal.</p><p>Includes decisions after [month year]. Find details of <a rel=\"external\" href=\"http://www.employmentappeals.gov.uk/Public/Search.aspx\">older cases.</a></p>",
```

Comment:
This copy came from the copy editor. We don't know the exact month yet. I'll put a specific month and year now. It can be changed later if needed.